bin/sdl.cc

```diff
@@ -524,9 +519,6 @@ void run_sdl()
     std::cout << "Time for event creation = " << timeForEventCreation << " ms\n";
     printTimingInformation(timevec, full_elapsed, avg_elapsed);
 
-    SDL::cleanModules();
-    SDL::freeEndCapMapMemory();
```

Comment:
This happens automatically now, because they are alpaka buffers?

Reply:
Well for both I use shared ptr's now which are freed when they go out of scope. Since these two things are both in global scope right now they get freed at the very end automatically. Although the modules have buffers within the object, and those also get freed at the end when the shared ptr gets freed.

Reply:
Got it, thanks! I still don't like the global scope but let's deal with it in another PR.